superset/assets/src/dashboard/components/Dashboard.jsx

```diff
@@ -17,8 +17,10 @@
  * under the License.
  */
 import React from 'react';
+import { Modal } from 'react-bootstrap';
 import PropTypes from 'prop-types';
 import { t } from '@superset-ui/translation';
+import $ from 'jquery';
```

Comment:
Please do not add new jquery usage.

Reply:
yeah this should be a blocker

Reply:
You prob need a div ref and https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/focus